### ISSUE TITLE
Specific Volume Keys Do Not Bind from External Keyboard

### ISSUE THREAD

**Issue by pyxtess** [MonitorControl/MonitorControl#1512]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

My Corsair K55 external keyboard has volume keys which work on isolated audio outputs (external speakers, headphones, monitor speakers, the macbook itself). 

When attempting to bind these keys in the Keyboard settings, they do not register as though no keys were selected.

### Steps to reproduce

1. Have an external keyboard with specific volume buttons — Volume up, Volume down, & Mute.
2. Go into the Monitor Control Settings and view the Keyboard tab.
3. In the Volume Control section, select on of the three options to bind key and tap respective volume key.

If the bug persists, the software will act as though no keys were pressed.

### Expected behavior

When rebinding keys, pressing the external keyboard's volume key bind and shows the visual text/id in the prompt.

### Anything else?

_Thoughts: My best guess that these specific keys might be register differently and is why they aren't being picked up by the software. I haven't tested this theory though, and need to see how these buttons are called by the computer because, whatever Corsair did, does get picked up by the Mac as a volume control button._ 

### Environment Information (please complete the following information)

```markdown
- macOS version: Sonoma 14.2.1
- Mac model: MacBook Pro (14-inch, 2021)
- MonitorControl version: v4.2.0
- Monitor(s): Acer CB272K; Samsung S34CG50; MacBook Pro (14-inch, 2021) Screen
- Apple Silicon/M1 (yes or no): yes; M1 Max
```

**Comment by waydabber** [comment]
Yes, these keys might be specially handled. Do they need Corsair's software for this feature to work? I am unfamiliar with the K55 but I don't think having volume keys assigned to different audio outputs is a standard macOS feature.
